engines/director/castmember/sound.cpp

```diff
@@ -79,9 +79,6 @@ void SoundCastMember::load() {
 		if (ci) {
 			Common::String filename = ci->fileName;
 
-			if (!ci->directory.empty())
```

Comment:
It was originally added in order to fix https://bugs.scummvm.org/ticket/13307  Is that still working?

Reply:
It is not, evidently I'll have to do more testing

Reply:
It's working for both Klondike Gold and Bob Winkle Solves Life's Greatest Mysteries